Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -756,6 +582,200 @@ public void EnableManager(Type type, string managerName)
             return (T)manager;
         }
 
+        #endregion Individual Manager Management
+
+        #region Multiple Managers Management
```

Comment:
Idk why, but Managers Management made me chuckle a little.

What about `Multiple Manager Utilities`?

Reply:
rofl, now we are strategizng on #if names :P  
Sure, I'll quickly change that so we can progress.

Reply:
😋